mitosheet/mitosheet/enterprise/mito_config.py

```diff
@@ -131,6 +135,11 @@ def get_support_email(self) -> str:
             return DEFAULT_MITO_CONFIG_SUPPORT_EMAIL
         return self.mec[MITO_CONFIG_SUPPORT_EMAIL]
 
+    def get_display_tours(self) -> bool:
+        if self.mec is None or self.mec[MITO_CONFIG_DISPLAY_TOURS] is None:
+            return True
+        return False if self.mec[MITO_CONFIG_DISPLAY_TOURS] == 'False' else True
```

Comment:
I think we should have a helper function called `is_env_variable_set_to_true`. It should test the variable against:
1. 1 and 0
2. True and false (upper and lower)
3. t and f

Generally, people will put weird shit in this. We should accept the most common inputs